src/store/account-details/account-details.thunks.ts

```diff
@@ -272,7 +273,10 @@ function fetchExits(tokenId: Token["id"]): AppThunk {
     } = getState();
     if (wallet !== undefined) {
       return CoordinatorAPI.getExits(wallet.hermezEthereumAddress, true, tokenId)
-        .then((exits: Exits) => dispatch(accountDetailsActions.loadExitsSuccess(exits)))
+        .then((exits: Exits) => {
+          dispatch(globalThunks.recoverPendingDelayedWithdrawals(exits));
```

Comment:
Shouldn't this be an account details thunk?

Reply:
It's global like `addPendingDelayedWithdraw` because it's used in home and account details